controller/lib/core/pinch_valve.cpp

```diff
@@ -0,0 +1,168 @@
+/* Copyright 2020, RespiraWorks
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "pinch_valve.h"
+#include "stepper.h"
+
+// These constants define various properties of the pinch
+// value and how we control it.  As the mechanical design
+// of the value continues to evolve these constants may
+// need to change.  Once things stabilize however, we should
+// be able to come up with a good set of values here that
+// won't need further modification.
+
+// Amplitude of the power level used for homing
```

Comment:
period

Reply:
I think amplitude is more accurate and more consistent with the terminology I use in my stepper driver.  Period is sort of right, but these settings don't directly set the period of the PWM signal (that's dependent on stepper position), they scale it.
If you watch the output to the motor (current or voltage) during motion you should see something roughly sinusoidal with an amplitude that corresponds to these settings.  That's where the name comes from.

Reply:
Sorry, I just meant, the sentence could use a period at the end

Reply:
LoL - right!  Period in the other sense actually made some sense though and I spent some time trying to decide if amplitude or period was actually better.